backend/src/models/userCreds.js

```diff
@@ -0,0 +1,17 @@
+const mongoose = require('mongoose');
+const bcrypt = require('bcryptjs');
```

Comment:
This import should probably be removed, and if we aren't using bcryptjs anywhere else we should remove it from the `package.json`